### DISCUSSION TITLE
Visual Basic Projects incorrectly included in .gitignore  template

### DISCUSSION THREAD

**Discussion by tomxp411** [community/community#112627] [Repositories]
### Select Topic Area

Bug

### Body

tl;dr: the Visual Studio .gitignore template should not include `*.vbp` as an "ignore" file type.

Full description:

I recently created a new Github repo for some Visual BASIC 6 code. I selected the Visual Studio template for .gitignore, and I later discovered that none of the .VBP files were uploaded to the repository.

When I looked in the .gitignore, I found this:

```
# Visual Studio 6 auto-generated project file (contains which files were open etc.)
*.vbp
```

This is _incorrect_. The VBP file is an important part of the project, and you cannot build a project without it. This is essentially the Visual Studio 6 version of a Makefile, which you would never leave out of a repository.

All of the other Visual Studio 6 ignores seem fine, but the `.vbp` should not be in this template.
